docs/elements.md

```diff
@@ -1906,7 +1906,7 @@ Displays boilerplate HTML from templates in a reusable way.
 #### Sample element
 
 ```html
-<pl-template file-name="outer_template.mustache" subdirectory="templates">
+<pl-template file-name="templates/outer_template.mustache" directory="serverFilesCourse">
   <pl-variable name="is-open">True</pl-variable>
   <pl-variable
```

Comment:
(3 lines below, but I can't comment on GitHub): there is no such thing as `serverFilesQuestion`.

Reply:
Done here https://github.com/PrairieLearn/PrairieLearn/pull/8434/commits/1bc546cef6db9bad95776c83a59fc83da54d36ad